### ISSUE TITLE
NullReferenceException in VersionedFieldIsChanged method

### ISSUE THREAD

**Issue by BerserkerDotNet** [kamsar/Unicorn#49]
Hi Kam,

I got a NullReferenceException while synchronizing items.
Exception:
```C#
[U] master:/sitecore/templates/System/Language (f68f13a6-3395-426a-b9a1-fa2dc60d94eb)
ERROR: Object reference not set to an instance of an object. (System.NullReferenceException)
at Unicorn.Evaluators.DefaultSerializedAsMasterEvaluatorLogger.VersionedFieldIsChanged(IItemData serializedItemData, IItemVersion version, Guid fieldId, String serializedValue, String sourceValue) 
at Unicorn.Evaluators.SerializedAsMasterEvaluator.<>c__DisplayClass12.b__6(ISerializedAsMasterEvaluatorLogger log) 
at Unicorn.Logging.DeferredLogWriter`1.ExecuteDeferredActions(TLog log) 
at Unicorn.Evaluators.SerializedAsMasterEvaluator.EvaluateUpdate(IItemData sourceItem, IItemData targetItem) 
at Unicorn.Loader.SerializationLoader.DoLoadItem(IItemData serializedItemData, IConsistencyChecker consistencyChecker) 
at Unicorn.Loader.SerializationLoader.LoadTree(IItemData rootItemData, IDeserializeFailureRetryer retryer, IConsistencyChecker consistencyChecker) 
at Unicorn.Loader.SerializationLoader.LoadAll(IItemData[] rootItemsData, IDeserializeFailureRetryer retryer, IConsistencyChecker consistencyChecker, Action`1 rootLoadedCallback) 
at Unicorn.SerializationHelper.SyncTree(IConfiguration configuration, Action`1 rootLoadedCallback, IItemData[] roots) 
at Unicorn.ControlPanel.SyncConsole.Process(IProgressStatus progress)
```

What I did is I simply updated project on my local machine it went fine, no errors.
When I pushed my changes to a dev server it failed to "Sync" items with this error.
I made a clean "Serialization" on dev, and turns out that there is only one difference in failing files:
![sc_lang_diff](https://cloud.githubusercontent.com/assets/1640125/9601101/0cd64e1c-50a8-11e5-9a02-d1b522e0ed49.png)

This issue appears not only in Language item it appears in other items not created by me (Sitecore items).
From the de-compiler I can see that there are not a lot of places for NullRef, 
![possible_errors](https://cloud.githubusercontent.com/assets/1640125/9601178/98e31cf0-50a8-11e5-899f-f761fe61ec00.png)

Please let me know if you need more info.
Thanks.

**Comment by kamsar** [comment]
Thanks, I'll get that fixed

**Comment by kamsar** [comment]
This is fixed in 3.0b5
